### ISSUE TITLE
v25 can not search for viruses / cannot open

### ISSUE THREAD

**Issue by Smiril** [bitcoin/bitcoin#28517]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current behaviour

hello the versions from https://bitcoincore.org/bin/bitcoin-core-25.0/ for macOS arm64 dmg can't opened.  
no signature in project !

### Expected behaviour

open the dmg is smooth but after installing cannot opened by failing signature

not signed for macOS from developer

### Steps to reproduce

just install and open

### How did you obtain Bitcoin Core

Pre-built binaries

### What version of Bitcoin Core are you using?

v25

### Operating system and version

MacOS Ventura 13.5.2

### Machine specifications

arm64 M1 MacBook Pro

**Comment by hebasto** [comment]
Mind providing a screenshot?

**Comment by Smiril** [comment]
https://ibb.co/1K7Lcdc

**Comment by Smiril** [comment]
> Mind providing a screenshot?

https://ibb.co/1K7Lcdc

**Comment by Smiril** [comment]
> Mind providing a screenshot?

https://ibb.co/Kq93H8C

the tar.gz also

**Comment by hebasto** [comment]
> ### Is there an existing issue for this?
> 
>     * [x]  I have searched the existing issues

https://bitcoin.stackexchange.com/questions/110478/macos-monterey-refuses-to-open-bitcoin-core-app-because-apple-cannot-check-it-fo

Did you try that answer?

**Comment by Smiril** [comment]
yes I´ve test it twice

I build from source and I can open ... because I got some apple developer account 
on this version that I build I got some warning "not for mining and trading application use at your own risk"

the downloaded version is also different than the build version 
See:
https://ibb.co/vJh3nQJ

**Comment by hebasto** [comment]
> the downloaded version is also different than the build version

Did you use the reproducible way to build binaries using [Guix](https://github.com/bitcoin/bitcoin/blob/master/contrib/guix/README.md)?

**Comment by Smiril** [comment]
I use the way doc/build-osx.md

**Comment by hebasto** [comment]
> Will this be fixed in 26.0 by the zip?

No. Notarization is required. For details, please see https://github.com/bitcoin/bitcoin/issues/15774.

**Comment by Smiril** [comment]
I´ll upgraded macOS and now it works ... Ventura 13.6
and I click on "show package source"(first under open in context menu) to inspect it
many resources not in the build version 
the prebuild version opens after inspecting on doubleclick
